Makefile

```diff
@@ -3,29 +3,29 @@ TRAVIS_BUILD_NUMBER?=debug0
 
 .PHONY: coverage get test clean
 
-all: get coverage AWSnycast
+all: _vendor coverage AWSnycast
 
-AWSnycast: *.go */*.go
-	go build -a -tags netgo -ldflags '-w' .
-	strip AWSnycast
+_vendor: Gomfile
```

Comment:
GOPATH can have multiple elements:

GOPATH=/Users/tdoran/go:/usr/local/shared_go

for example. I've never seen anything/anyone doing this 'in the wild', but it's possible, so we'd need to split GOPATH etc.

I don't really have any issues with users having to run go get github.com/mattn/gom - as long as the workflow to get developing is fairly easy.

My actual plan to solve this problem is just to add a 'Developing AWSnycast' section to the README, which can have some notes about the expectations around GOPATH and note that you need to manually install gom yourself.

I think that as long as we clearly document the expectations, then, for the common case (i.e. where people who already know some go are likely to hack on it, rather than absolute beginners) that we're better off sticking to the standard locations for everything (and expecting folks to set up their environment the 'normal' Go way).